routing_deep_dive.prolific

```diff
@@ -0,0 +1,130 @@
+Routing Control Plane Overview
+
+### What?
+In previous stories you pushed apps and created routes. In this set of stories you are going to learn the "behind-the-scenes" on how routes are propagated.
+
+Here is the entire overview. Each step marked with a ✨will be explained in more detail in a story in this track.
+
+**When an app dev maps a route**
+1. The app dev maps a route to an app using CAPI.
+1. CAPI sends this information to Diego BBS.
+1. The Route Emitter on each Diego Cell polls Diego BBS for updated information about the apps on its Cell.
+1. ✨ The Route Emitter continually sends the route registration information it gets from the Diego BBS and the IP and port of the app to the GoRouter via NATS.
+1. ✨ The GoRouter keeps a mapping of routes -> ip:ports in a routes table, which is consulted when someone curls the route.
+
+### How?
+The following stories will look at how many components (CAPI, Diego BBS, Route Emitter, Nats, GoRouter) work together to make routes work.
+
+### Expected Result
+You can talk about route propagation at a high level.
+
+### Resources
+[CF Routes and Domains](https://docs.cloudfoundry.org/devguide/deploy-apps/routes-domains.html)
+
+
+L: routing
+---
+
+Subscribe to Route Emitter NATS Messages
+
+### What?
+There is one Route Emitter per Diego Cell and its job is to... emit routes.
+
+The Route Emitter tracks the desired route mappings for all of the apps it shares a Cell with. When a change is detected, the Route Emitter emits route registration and unregistration messages to the GoRouter via the NATS message bus. Even when no change is detected, the Route Emitter will periodically emit the current state of the world as a kind of heartbeat.
+
+For this story, let's look at the messages that the Route Emitter is publishing via NATS. Subscribing to these NATs messages can be a helpful debugging technique when it seems like routes are not being registered properly.
+
+### How?
+0. Bosh ssh onto a Diego Cell
+0. Download ruby and the NATS gem
+    ```
+    apt-get install -y ruby ruby-dev
+    gem install nats
+    ```
+0. Get the NATS username, password, and server address
+    ```
+    cat /var/vcap/jobs/route_emitter/config/route_emitter.json | jq . | grep nats
+    ```
+0. Use the nats gem to connect to nats:
+```nats-sub "*.*" -s nats://NATS_USERNAME:NATS_PASSWORD@NATS_ADDRESS```
```

Comment:
I see you did multi-line previously but this one is single line was that intentional or a typo?